### PR TITLE
Frontend changes

### PR DESCRIPTION
**Pull request by Shehrozkashif**
The buttons location has been changed and code format has been improved

### CODE REVIEW

File: frontend/web/index.html [hunk 1]
```diff
@@ -4,109 +4,24 @@
...
 	<title>ArcheV </title>
-
 	<!-- Compressed Styles -->
 	<link href="css/slides.min.css?41331" rel="stylesheet" type="text/css">
-
-	<!-- Custom Styles -->
-	<!-- <link href="css/custom.css" rel="stylesheet" type="text/css"> -->
-
 	<!-- jQuery 3.3.1 -->
 	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
```
Comment (5hayanB) [inline frontend/web/index.html:12]: Why are you using jquery?

File: frontend/web/index.html [hunk 2]
```diff
@@ -4,109 +4,24 @@
...
-
 	<!-- Compressed Scripts -->
 	<script src="js/slides.min.js?41331" type="text/javascript"></script>
-
-	<!-- Custom Scripts -->
-	<!-- <script src="js/custom.js" type="text/javascript"></script> -->
-
+  <script src="js/select_button.js" defer></script>
```
Comment (5hayanB) [inline frontend/web/index.html:15]: Indentation out of line

File: frontend/web/index.html [hunk 3]
```diff
@@ -4,109 +4,24 @@
...
+  <script src="js/backend.js" defer></script>
+  <link rel="stylesheet" href="css/styles.css">
 	<!-- Fonts and Material Icons -->
 	<link rel="stylesheet" as="font" href="https://fonts.googleapis.com/css?family=Roboto:100,300,400,500,600,700|Material+Icons"/>
```
Comment (5hayanB) [inline frontend/web/index.html]: Download the fonts instead of using an api for it

File: frontend/web/index.html [hunk 4]
```diff
@@ -4,109 +4,24 @@
...
-                if (files.length > 0) {
-                    let folderPath = files[0].webkitRelativePath.split('/')[0];
-                    $('#llm-path').val(folderPath);
-                }
-            });
-        });
-    });
-</script>
-
 <!-- Fonts and Material Icons -->
```
Comment (5hayanB) [inline frontend/web/index.html]: Indentation out of line. Download the fonts instead of using an api for it

File: frontend/web/index.html [hunk 5]
```diff
@@ -4,109 +4,24 @@
...
 <link rel="stylesheet" as="font" href="https://fonts.googleapis.com/css?family=Roboto:100,300,400,500,600,700|Material+Icons"/>
 </head>
- 
-
-
-
 </head>
 <body class="slides chain simplifiedMobile animated">
-		
 <!-- SVG Library -->
```
Comment (5hayanB) [inline frontend/web/index.html:22]: Add indentation

File: frontend/web/index.html [hunk 6]
```diff
@@ -4,109 +4,24 @@
...
...9.04 7.36 4.92 0 9.04-2.88 9.04-7.76 0-4.8-4-5.92-7.76-6.96-1.76-.52-3.4-1.2-3.4-2.2 0-.6.48-1.48 1.88-1.48 1.96 0 2.04 1.2 2.08 1.68h6.52zm2.222 15.96h6.64v-29.6h-6.64v29.6zm9.662-24.56h6.64v-5.04h-6.64v5.04zm0 24.56h6.64v-22.2h-6.64v22.2zm32.782-29.6h-6.64v9.28c-.72-.72-2.6-2.64-6.52-2.64-5.64 0-11 4.28-11 11.8 0 6.68 4.4 11.88 11.12 11.88 4.48 0 6.08-2.2 6.72-3.12v2.4h6.32v-29.6zm-17.52 18.4c0-2.56 1.8-5.56 5.64-5.56 1.56 0 2.96.56 3.96 1.56 1 .96 1.64 2.32 1.64 3.92.08 1.64-.52 3.08-1.56 4.12s-2.52 1.68-4.12 1.68c-3.12 0-5.56-2.28-5.56-5.68v-.04zm42.502 2.4c.52-4.08-.32-7.64-3.12-10.64-2.08-2.2-5-3.52-8.4-3.52-6.76 0-11.64 5.72-11.64 11.92 0 6.6 5.4 11.76 11.76 11.76 2.28 0 4.48-.68 6.32-2 1.88-1.28 3.44-3.2 4.52-5.68h-6.8c-.8 1.16-1.92 2.08-4.04 2.08-2.6 0-4.84-1.56-5.12-3.92h16.52zm-16.44-5.04c.16-1.04 1.52-3.52 4.96-3.52s4.8 2.48 4.96 3.52h-9.92zm34.502-2.12c-.56-5.84-5.6-7-8.52-7-4.6 0-8.6 2.92-8.6 7.52 0 3 2.4 4.88 5.28 5.8 4.24 1.64 5.88 1.84 5.88 3.36 0 1.08-1.2 1.72-2.32 1.72-.28 0-2.24 0-2.52-2.04h-6.6c.6 5.84 5.68 7.36 9.04 7.36 4.92 0 9.04-2.88 9.04-7.76 0-4.8-4-5.92-7.76-6.96-1.76-.52-3.4-1.2-3.4-2.2 0-.6.48-1.48 1.88-1.48 1.96 0 2.04 1.2 2.08 1.68h6.52z"/></symbol>
```
Comment (5hayanB) [inline frontend/web/index.html:24]: Make it 4 spaces of indentation

File: frontend/web/index.html [hunk 7]
```diff
@@ -127,54 +42,20 @@
   <symbol id="sound-on" viewBox="0 0 18 18"><path d="M8.5,0.1C8.1-0.1,7.7,0,7.4,0.2L3.7,3H2C0.9,3,0,3.9,0,5v6c0,1.1,0.9,2,2,2h1.7l3.7,2.8C7.6,15.9,7.8,16,8,16 c0.2,0,0.3,0,0.4-0.1C8.8,15.7,9,15.4,9,15V1C9,0.6,8.8,0.3,8.5,0.1z M7,13l-2.4-1.8C4.4,11.1,4.2,11,4,11l-2,0l0-6h2 c0.2,0,0.4-0.1,0.6-0.2L7,3V13z M11.7,9.9l0.7,1.9C13.9,11.2,15,9.7,15,8c0-1.7-1.1-3.2-2.7-3.8l-0.7,1.9C12.5,6.4,13,7.2,13,8C13,8.9,12.5,9.6,11.7,9.9z M12.2,1.1l-0.3,2C14.3,3.5,16,5.6,16,8s-1.8,4.5-4.2,4.9l0.3,2C15.6,14.3,18,11.4,18,8C18,4.6,15.6,1.7,12.2,1.1z"/></symbol>
```
Comment (5hayanB) [inline frontend/web/index.html:39]: Make it 4 spaces of indentation

File: frontend/web/index.html [hunk 8]
```diff
@@ -127,54 +42,20 @@
...
   <symbol id="sound-off" viewBox="0 0 18 18"><path d="M15.9,8l1.8-1.8c0.4-0.4,0.4-1,0-1.4s-1-0.4-1.4,0l-1.8,1.8l-1.8-1.8c-0.4-0.4-1-0.4-1.4,0s-0.4,1,0,1.4L13.1,8l-1.8,1.8 c-0.4,0.4-0.4,1,0,1.4c0.2,0.2,0.5,0.3,0.7,0.3s0.5-0.1,0.7-0.3l1.8-1.8l1.8,1.8c0.2,0.2,0.5,0.3,0.7,0.3s0.5-0.1,0.7-0.3 c0.4-0.4,0.4-1,0-1.4L15.9,8z M8.5,0.1C8.1-0.1,7.7,0,7.4,0.2L3.7,3H2C0.9,3,0,3.9,0,5v6c0,1.1,0.9,2,2,2h1.7l3.7,2.8C7.6,15.9,7.8,16,8,16 c0.2,0,0.3,0,0.4-0.1C8.8,15.7,9,15.4,9,15V1C9,0.6,8.8,0.3,8.5,0.1z M7,13l-2.4-1.8C4.4,11.1,4.2,11,4,11l-2,0l0-6h2 c0.2,0,0.4-0.1,0.6-0.2L7,3V13z"/></symbol>
```
Comment (5hayanB) [inline frontend/web/index.html:40]: Make it 4 spaces of indentation

File: frontend/web/index.html [hunk 9]
```diff
@@ -127,54 +42,20 @@
...
   <!-- social -->
```
Comment (5hayanB) [inline frontend/web/index.html:41]: Make it 4 spaces of indentation

File: frontend/web/index.html [hunk 10]
```diff
@@ -127,54 +42,20 @@
...
-
-  <symbol id="dribbble" viewBox="0 0 24 24"><path d="M12 0c-6.7 0-12 5.3-12 12s5.3 12 12 12 12-5.3 12-12-5.3-12-12-12zm7.9 5.7c1.3 1.7 2.1 3.9 2.3 6.1-.4-.1-2.4-.4-4.7-.4-.8 0-1.5 0-2.3.1 0-.1-.1-.3-.3-.5l-.7-1.5c3.7-1.4 5.3-3.4 5.7-3.8zm-7.9-3.8c2.5 0 4.9.9 6.7 2.5-.3.4-1.9 2.3-5.2 3.6-1.6-2.9-3.3-5.3-3.7-5.9.6-.1 1.4-.2 2.2-.2zm-4.4 1c.4.6 2.1 3 3.7 5.8-4.4 1.2-8.2 1.2-9.2 1.2h-.1c.8-3.1 2.9-5.6 5.6-7zm-5.7 9.1v-.3h.3c1.2 0 5.6-.1 10.1-1.5l.8 1.6c-.1 0-.3 0-.4.1-5.1 1.6-7.9 6-8.3 6.7-1.6-1.7-2.5-4.1-2.5-6.6zm10.1 10.1c-2.3 0-4.4-.8-6.1-2.1.3-.5 2.4-4.4 7.9-6.3 1.3 3.6 2 6.7 2.1 7.6-1.2.6-2.6.8-3.9.8zm5.7-1.8c-.1-.8-.7-3.6-2-7.1.7-.1 1.3-.1 2-.1 2.1 0 3.7.4 4.1.5-.3 2.8-1.8 5.2-4.1 6.7z"/></symbol>
-
   <symbol id="facebook" viewBox="0 0 24 24"><path d="M24 1.3v21.3c0 .7-.6 1.3-1.3 1.3h-6.1v-9.3h3.1l.5-3.6h-3.6v-2.2c0-1.1.3-1.8 1.8-1.8h1.9v-3.2c-.3 0-1.5-.1-2.8-.1-2.8 0-4.7 1.7-4.7 4.8v2.7h-3.1v3.6h3.1v9.2h-11.5c-.7 0-1.3-.6-1.3-1.3v-21.4c0-.7.6-1.3 1.3-1.3h21.3c.8 0 1.4.6 1.4 1.3z"/></symbol>
```
Comment (5hayanB) [inline frontend/web/index.html:42]: Make it 4 spaces of indentation

File: frontend/web/index.html [hunk 11]
```diff
@@ -127,54 +42,20 @@
...
-   
-  <symbol id="fb-like" viewBox="0 0 20 20"><path d="M0 8v12h5v-12h-5zm2.5 10.8c-.4 0-.8-.3-.8-.8 0-.4.3-.8.8-.8s.8.3.8.8c0 .4-.4.8-.8.8zm3.5-.8h9.5c1.1 0 1.7-1 1.7-1.7 0-.3-.4-1-.4-1 1.4-.3 1.7-1.2 1.7-1.7-.1-.5-.3-.9-.5-1 1-.4 1.5-1.1 1.4-1.9-.1-.8-1-1.5-1-1.5 1-.6.9-1.5.9-1.5-.3-1.3-1.5-1.7-1.7-1.7h-5.6s.3-.5.3-2.4-1.3-3.6-2.6-3.6c0 0-.7.1-1 .3v3.5l-2.7 4.4v9.8z"/></symbol>
-  
-  <symbol id="googlePlus" viewBox="0 1 24 24"><path d="M7.8 13.5h4.6c-.6 2-2.5 3.4-4.6 3.4-2.7 0-4.9-2.2-4.9-4.9s2.2-4.9 4.9-4.9c1.1 0 2.1.3 3 1l1.8-2.4c-1.4-1.1-3-1.6-4.8-1.6-4.3 0-7.9 3.5-7.9 7.9s3.5 7.9 7.9 7.9 7.9-3.5 7.9-7.9v-1.5h-7.9v3zM21.7 11v-2.2h-2v2.2h-2.2v2h2.2v2.2h2v-2.2h2.2v-2z"/></symbol>
-  
   <symbol id="instagram" viewBox="0 0 20 20"><circle cx="10" cy="10" r="3.3"/><path d="M13,0H7C2.2,0,0,2.2,0,7v6c0,4.8,2.1,7,7,7h6c4.8,0,7-2.2,7-7V7C20,2.2,17.9,0,13,0z M10,15.1c-2.8,0-5.1-2.3-5.1-5.1 S7.2,4.9,10,4.9s5.1,2.3,5.1,5.1S12.8,15.1,10,15.1z M15.3,5.9c-0.7,0-1.2-0.5-1.2-1.2c0-0.7,0.5-1.2,1.2-1.2s1.2,0.5,1.2,1.2 C16.5,5.3,16,5.9,15.3,5.9z"/></symbol>
```
Comment (5hayanB) [inline frontend/web/index.html:43]: Make it 4 spaces of indentation

File: frontend/web/index.html [hunk 12]
```diff
@@ -127,54 +42,20 @@
...
-  
-  <symbol id="pinterest" viewBox="0 0 24 24"><path d="M5.9 13.9c1.2-2-.4-2.5-.6-4-1-6.1 7.1-10.2 11.4-6 2.9 2.9 1 12-3.7 11-4.6-.9 2.2-8.1-1.4-9.5-3-1.1-4.6 3.6-3.2 5.9-.8 4-2.5 7.7-1.8 12.7 2.3-1.7 3.1-4.8 3.7-8.1 1.2.7 1.8 1.4 3.3 1.5 5.5.4 8.6-5.4 7.8-10.7-.7-4.7-5.5-7.1-10.6-6.6-4.1.4-8.1 3.7-8.3 8.3-.1 2.8.7 4.9 3.4 5.5z"/></symbol>
-  
-  <symbol id="stumbleupon" viewBox="0 0 24 24"><path d="M13.3 9.6l1.6.8 2.5-.8v-1.4c0-3-2.4-5.4-5.4-5.4s-5.4 2.4-5.4 5.4v7.5c0 .7-.6 1.3-1.3 1.3s-1.3-.6-1.3-1.3v-3.2h-4v3.2c0 3 2.4 5.4 5.4 5.4s5.4-2.4 5.4-5.4v-7.5c0-.7.6-1.3 1.3-1.3s1.3.6 1.3 1.3l-.1 1.4zm6.6 2.9v3.2c0 .7-.6 1.3-1.3 1.3s-1.3-.6-1.3-1.3v-3.2l-2.5.8-1.6-.8v3.2c0 3 2.4 5.4 5.4 5.4s5.4-2.4 5.4-5.4v-3.2h-4.1z"/></symbol>
-  
   <symbol id="twitter" viewBox="0 1 24 23"><path d="M21.5 7.6v.6c0 6.6-5 14.1-14 14.1-2.8 0-5.4-.8-7.6-2.2l1.2.1c2.3 0 4.4-.8 6.1-2.1-2.2 0-4-1.5-4.6-3.4.3.1.6.1.9.1.5 0 .9-.1 1.3-.2-2.1-.6-3.8-2.6-3.8-5 .7.4 1.4.6 2.2.6-1.3-.9-2.2-2.4-2.2-4.1 0-.9.2-1.8.7-2.5 2.4 3 6.1 5 10.2 5.2-.1-.4-.1-.7-.1-1.1 0-2.7 2.2-5 4.9-5 1.4 0 2.7.6 3.6 1.6 1-.3 2.1-.7 3-1.3-.4 1.2-1.1 2.1-2.2 2.7 1-.1 1.9-.4 2.8-.8-.6 1.1-1.4 2-2.4 2.7z"/></symbol>
```
Comment (5hayanB) [inline frontend/web/index.html:44]: Make it 4 spaces of indentation

File: frontend/web/index.html [hunk 13]
```diff
@@ -127,54 +42,20 @@
...
-  
-  <symbol id="youtube" viewBox="0 0 24 24"><path d="M23.6 6.3c-.3-1.2-1.4-2.2-2.6-2.3-3-.3-6-.3-9-.3s-6 0-9 .3c-1.2.1-2.3 1.1-2.6 2.3-.4 1.8-.4 3.8-.4 5.7 0 1.9 0 3.9.4 5.7.3 1.2 1.4 2.2 2.6 2.3 3 .3 6 .3 9 .3s6 0 9-.3c1.3-.1 2.3-1.1 2.6-2.4.4-1.7.4-3.7.4-5.6 0-1.9 0-3.9-.4-5.7zm-14.1 9v-6.6l6.5 3.3-6.5 3.3z"/></symbol>
-
 </svg>
```
Comment (5hayanB) [inline frontend/web/index.html]: Make it 4 spaces of indentation

File: frontend/web/index.html [hunk 14]
```diff
@@ -127,54 +42,20 @@
...
-
 <!-- Navigation -->
```
Comment (5hayanB) [inline frontend/web/index.html:46]: Make it 4 spaces of indentation

File: frontend/web/index.html [hunk 15]
```diff
@@ -127,54 +42,20 @@
...
 <nav class="side pole">
   <div class="navigation">
     <ul></ul>
   </div>
 </nav>
-
 <!-- Panel Top #08 -->
```
Comment (5hayanB) [inline frontend/web/index.html:52]: Make it 4 spaces of indentation

File: frontend/web/js/backend.js [hunk 16]
```diff
@@ -0,0 +1,24 @@
+function selectPath() {
+    window.pywebview.api.select_path().then(path => {
+        document.getElementById('llm-path').value = path;
+    });
+}
+
+
+function submitForm(event) {
```
Comment (5hayanB) [inline frontend/web/js/backend.js:8]: Why are you using events?

File: frontend/web/js/backend.js [hunk 17]
```diff
@@ -0,0 +1,24 @@
...
+    event.preventDefault();
```
Comment (5hayanB) [inline frontend/web/js/backend.js:9]: There is no need for this.

File: frontend/web/js/backend.js [hunk 18]
```diff
@@ -0,0 +1,24 @@
+function selectPath() {
+    window.pywebview.api.select_path().then(path => {
```
Comment (5hayanB) [inline frontend/web/js/backend.js:2]: Where is this function even defined?

File: frontend/web/js/backend.js [hunk 19]
```diff
@@ -0,0 +1,24 @@
...
+    const gpuLayers = document.getElementById('gpu-layers').value;
+    const contextNumber = document.getElementById('context-number').value;
+
+    const data = {
+        llm_path: llmPath,
+        gpu_layers: gpuLayers,
+        context_number: contextNumber
+    };
+
+    window.pywebview.api.submit_form(data).then(response => {
```
Comment (5hayanB) [inline frontend/web/js/backend.js:20]: Where is this `submit_form` function?

**Review by 5hayanB — CHANGES_REQUESTED** [review]
Please apply the requested changes

File: frontend/web/index.html (hunk 1 above)
Comment (Shehrozkashif) [reply frontend/web/index.html:12]: jquery is the part of template

**Pull request closed without merging**